canvas/base.go

```diff
@@ -19,47 +17,30 @@ type baseObject struct {
 	Hidden   bool          // Is this object currently hidden
 
 	min fyne.Size // The minimum size this object can be
-
-	propertyLock sync.RWMutex
```

Comment:
I have not yet familiar with the existing `propertyLock` design yet. What's the rationale that we could remove it entirely?

Reply:
> My understanding is that this lock is just silencing the race condition instead of addressing it.

I am strongly against the view of "lock is just silencing the race". Locks are considered a memory barrier to guarantee a single read/write op at a time, which solves the race condition by definition.

Reply:
Ah, of course, the old code does not solve race conditions entirely, but the initial question was: it is also not well explained why removing it can solve the race condition entirely.

Reply:
It is not removing it that solve the race condition, it is the copy that does solve the problem.